Client/Frontend/Intro/IntroViewControllerV2.swift

```diff
@@ -0,0 +1,108 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import UIKit
+import SnapKit
+import Shared
+import Leanplum
+
+class IntroViewControllerV2: UIViewController {
+    private lazy var welcomeCard: IntroScreenWelcomeViewV2 = {
+        let welcomeCardView = IntroScreenWelcomeViewV2()
+        welcomeCardView.contentMode = .scaleAspectFit
+        welcomeCardView.clipsToBounds = true
+        welcomeCardView.tag = 0
```

Comment:
same tag as syncCardView.tag, so same tag numbers is odd, but also not sure if it is used?

Reply:
- removed unwanted code